oonib.conf.example

```diff
@@ -7,11 +7,6 @@ main:
     bouncer_file: data/bouncer.yaml
 
     logfile: null
-    database_uri: 'sqlite://oonib_test_db.db'
-    db_threadpool_size: 10
-    tor_binary: null
```

Comment:
We actually need the ability to specify a custom tor binary so we can use tor2web mode. I actually didn't realise that this had already been removed in the previous PR, but it should go back in.